### PR TITLE
Hide displaynames by default in fullstory

### PR DESCRIPTION
**Pull request by Discordius**

### CODE REVIEW

**Comment by jimrandomh** [comment]
While seeing names is probably too much, I do want some sense of whether a user is a core user or a peripheral user. What if we displayed karma, but bucketed? (Ie, display whether their karma is <20, 20-200, 201-2000, or 2001+?)

**Comment by Raemon** [comment]
I agree that "bucketed karma" is a decent approach, although think it makes most sense to get the basic version of this working at all, and shouldn't be blocking this PR.

File: packages/lesswrong/client/ga.js [hunk 1]
```diff
@@ -20,10 +20,11 @@ function googleTagManagerInit() {
 addInitFunction(googleTagManagerInit)
 
 const identifyFullStoryUser = (currentUser) => {
-  const { karma = 0, afKarma = 0, frontpagePostCount = 0, voteCount = 0, createdAt } = currentUser
-  const fullStoryData = { karma, afKarma, frontpagePostCount, voteCount, createdAt }
+  const { karma = 0, afKarma = 0, frontpagePostCount = 0, voteCount = 0, createdAt, username, displayName: lWDisplayName } = currentUser
```
Comment (Raemon) [inline packages/lesswrong/client/ga.js]: I'm confused about where lwDisplayName comes from

Comment (Discordius) [reply packages/lesswrong/client/ga.js]: This is the line where it is created

Comment (Discordius) [reply packages/lesswrong/client/ga.js]: That's how you rename variables in deconstructions. 

`const { a: b } = {a: "asdasd"}`

creates a new variable b = "asdasd"

**Comment by Discordius** [comment]
> While seeing names is probably too much, I do want some sense of whether a user is a core user or a peripheral user. What if we displayed karma, but bucketed? (Ie, display whether their karma is <20, 20-200, 201-2000, or 2001+?)

This still allows you to see the user displayname and their karma, it's now just a click away. You can still segment on it in Fullstory.

File: packages/lesswrong/client/ga.js (hunk 1 above)
Comment (jimrandomh) [reply packages/lesswrong/client/ga.js]: I quite dislike this syntax and would prefer splitting onto a separate line (ie `const lwDisplayName = currentUser.displayName`).

**Pull request merged**